package.json

```diff
@@ -38,9 +38,11 @@
   "dependencies": {
     "commander": "^2.9.0",
     "lodash": "^4.0.1",
+    "nlf": "^1.4.1",
     "node-fetch": "^1.3.3",
     "prepend-file": "^1.3.0",
     "promise": "^7.1.1",
+    "q": "^1.4.1",
```

Comment:
I don't think we really need two promise libraries. I realize the code being copied over was using `q`, but we should probably (in a follow-up PR if necessary) consolidate and convert the dependency compliance stuff to use the `promise` package instead.